merge master into live-based

luyajun0205 opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'live-based' into master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'master' into master-sync-work

Commit:
Merge pull request #24 from OPS-E2E-Prod/master-sync-work

Resolve syncing conflicts from master-sync-work to master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

v-susu-zz commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201807230658109656-39%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=elxwJL1Ca%2BM3ydrlauC3Lg7e1cUnyGKDJHbiK7yqbp4%3D&st=2018-07-23T06%3A57%3A10Z&se=2018-08-23T07%3A02%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

dend committed:
Update docfx.json

dend committed:
Merge pull request #951 from Azure/live

Live

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

troydai committed:
Azure CLI 2.0.43 release

azuresdkci committed:
Update CLI extensions available doc (#959)

* Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=409128688
https://github.com/Azure/azure-cli-extensions/commit/0ef3e90cbea52bd70c7ca08d41c59b7a035a5fc8

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Copyedit release notes.

Commit:
Fix search scope.

azuresdkci committed:
Update CLI extensions available doc (#961)

* Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=409629093
https://github.com/Azure/azure-cli-extensions/commit/89518d001832c619a2dad79b73892ed05e2dce01

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #962 from sptramer/search/fix-scope

Fix search scope.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #958 from troydai/patch-7

Azure CLI 2.0.43 release

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

v-maudel committed:
M29362: Removing possible extra word

Hello, @sptramer,
Translator has reported possible source content issue. 
"possible extra word at line 70"
Please review and merge the suggested proposed file change to avoid this error. If you make related fix in another PR, then share your PR number, so we can confirm and close this PR.
Many thanks in advance.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #975 from v-maudel/patch-2

M29362: Removing possible extra word

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azuresdkci committed:
Update CLI extensions available doc (#977)

* Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=413329030
https://github.com/Azure/azure-cli-extensions/commit/6381caf4f6fecbe8f38888c666512945d8911786

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Jaffacakes82 committed:
Update credential reset command

Update the credential reset command as per https://docs.microsoft.com/en-us/cli/azure/ad/sp/credential?view=azure-cli-latest#az-ad-sp-credential-reset

Commit:
Change docker -v option to be more correct. [Fixes #974]

sptramer committed:
Merge pull request #982 from sptramer/docker/fix-ssh

Change docker -v option to be more correct. [Fixes #974]

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #981 from Jaffacakes82/patch-1

Update credential reset command

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)
